Update How to use this User Guide section in UG (#255)
AY2223S1-CS2103T-F11-1/tp: docs/UserGuide.md

## 1.1 What is InterNUS?

> Welcome to InterNUS user guide!

Reviewer: ```suggestion
Welcome to the InterNUS user guide!
```

Track the internships you're interested in and all the relevant information -- the company, role, contact person and interview date. Awaiting a reply, or have you been accepted? With InterNUS, you can monitor and update the statuses of your internship applications effortlessly.

--------------------------------------------------------------------------------------------------------------------
# 2. How to use this User Guide?

> 1. For new user, proceed to our [Quick start](#3-quick-start)  

Reviewer: ```suggestion
1. For new users, proceed to our [Quick start](#3-quick-start).
```

> 1. For returning user, skip to our [Command summary](#command-summary) for a summary of our feature

Reviewer: ```suggestion
1. For returning users, skip to our [Command summary](#command-summary) for a summary of our features.
```

## 2.1 Icons and symbols
| Symbol               | Meaning                                                                                      |
|----------------------|----------------------------------------------------------------------------------------------|
| :information_source: | Additional information that may be useful to know when using InterNUS                        |
| :exclamation:        | Important information or warnings that you should take note of when using InterNUS           | 

> | `command`            | The highlighted words indicate commands that can be typed into the Command Input of InterNUS |

Reviewer: I think we shouldn't specify this since we use the `highlighted format` in many other cases.
```suggestion
```

## 2.2 Graphical User Interface (GUI) of InterNUS
![Ui](images/interNUS.png)

The UI component description:

| Components             | Purpose                                                    |
|------------------------|------------------------------------------------------------|
| Menu Bar               | Consists of File, Help and Color Theme                     |

> | Color Theme Button     | To display dropdown list to select between Light/Dark mode | 

Reviewer: ```suggestion
| Color Theme Button     | To switch between Light/Dark mode                          | 
```
I think we can be more concise here.

| Command Input          | To enter commands to be executed                           |

> | System Message Display | Display results of executed command                        |

Reviewer: ```suggestion
| System Message Display | Displays results of executed command                       |
```

   ![Ui](images/Ui.png)

1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>

1. Refer to the [Features](#features) below for details of each command.

--------------------------------------------------------------------------------------------------------------------

# 4. Features

## 4.1 Add Command

### 4.1.1 Adding a person: `add -p`

Adds a person to InterNUS. This person refers to the contact person you want to save. 
During internship application this person could be the hiring manager. 
After the application, this person could be the senior engineer who you want to keep contact at work.

Format: `add -p n/NAME [e/EMAIL] [p/PHONE_NUMBER] [t/TAG]…​ [l/LINK_INDEX] c/[COMPANY]`
* The link index (in add -p) refers to the index number shown in the internship list.
* The company refers to the company the contact person is working at.

<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
A person can have any number of tags (including 0). 
New added person will be added in current sorted order.
Only one contact person can be in-charge of one internship position.

> Phone number allows more than 2 digit without the need of any specific pattern.

Reviewer: ```suggestion
Phone number allows more than 2 digits without the need of any specific pattern.
```
I think this line can be moved up to the bulleted list above, as it's a constraint on a parameter and not really a "Tip".

### 4.9.1 Light and Dark Mode

> Clicking on Color Theme button in menu bar display a dropdown list to select between Light/Dark mode.

Reviewer: ```suggestion
Clicking on the Color Theme button in the Menu Bar displays a dropdown list to select between Light/Dark mode.
```
